charts/pega/values.yaml

```diff
@@ -462,8 +462,8 @@ hazelcast:
 
 # Stream (externalized Kafka service) settings.
 stream:
-  # Disabled by default, when enabled, your deployment no longer uses the "Stream" node type.
-  enabled: false
+  # Beginning with Pega Platform '23, enabled by default; when disabled, your deployment does not use a"Kafka stream service" configuration.
+  enabled: true
```

Comment:
Hi @girirajguptagithub 
Does this change compliant with the Zero down time requirement during upgrades? What would be the flow for the helm chart upgrades to this particular version?

Reply:
@kishorv10 :  Start with Infinity'23 externalization goal, the leadership has made decision to make external services as default deployment type. Infinity'23 has already baked these changes and now helm charts are made to be in-line with the same strategic direction. 

Yes, These changes are compliant with zero downtime upgrades provided the customers are upgrading from external to external.

Reply:
@anuragh27crony  What is the flow for embedded Kafka (stream tier) to external Kafka (to this version) upgrade? Does it also ZDT?

Reply:
Hi @kishorv10: Yes, it will be ZDT. We will be leveraging the similar approach used for cutty2cloudk for this as well.

Reply:
It is discussed that,  embedded Kafka (stream tier) to external Kafka (to this version) upgrade would be ZDT, and required client documentation will be updated in follow-up work.